icu4j/main/classes/core/src/com/ibm/icu/text/DisplayOptions.java

```diff
@@ -726,6 +726,22 @@ public final String getIdentifier() {
       return this.identifier;
     }
 
+
+    /**
+     * This an internal method.
+     *
+     * @return the lowercase CLDR keyword string for the grammatical case and {@code null} if the
+     * grammatical case is undefined.
+     * @deprecated ICU 72
+     */
+    public final String getIdentifierOrNull() {
+      if (this.equals(GrammaticalCase.UNDEFINED)) {
+        return null;
+      }
+
+      return this.identifier;
+    }
+
```

Comment:
Hopefully you don't need this helper, but if you do...
```suggestion
    /**
     * This an internal method.
     *
     * @return the lowercase CLDR keyword string for the grammatical case and {@code null} if the
     * grammatical case is undefined.
     *
     * @internal
     * @deprecated This API is ICU internal only.
     */
    public final String getIdentifierOrNull() {
        if (this.equals(GrammaticalCase.UNDEFINED)) {
            return null;
        }

        return this.identifier;
    }

```